doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -39,6 +44,28 @@ dashbuilder/navigation/readme.md
 VERSION
 ----
 
+
+If you want to export customized user created dashbuilder data as a ZIP file, do the following tasks:
+
+.. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+.. In *Export Wizard* panel, select the datasets which you want to be present in the ZIP file and click *Next*.
+.. Select the Pages which you want to be present in the ZIP file and click *Next*.
++
+The *Export Wizard* panel validates the selected datasets and pages. The summary of the datasets and pages is available on the panel.
++
+NOTE: By default, Navigation is always exported.
```

Comment:
Question - "By default" implies that the user can change this behavior. If this is the case, I think there needs to be an additional step. If the user can not change this, then I think the wording could be changed:

s/NOTE: By default, Navigation is always exported./NOTE: Navigation is always included in the exported ZIP file.

Reply:
Okay